decentraland/unity-renderer
# fix: Unexpected backpack behaviours

## What does this PR change?
This PR fixes this issue: https://github.com/decentraland/unity-renderer/issues/2595

The backpack was loosing all the COLLECTIBLES wearables each time we did this steps:

1. Open the backpack
2. Change your shoes
3. Save
4. Notice your avatar has been corretly modified
5. Open the backpack again (do this step quickly after the previous one)
6. Change tour shoes to another ones
7. Save
8. Notice your avatar has been corretly modified
9. Open the backpack again (do this step quickly after the previous one)
10. Notice your avatar preview has lost all your custom wearables (if you go to the COLLECTIBLES tab, you won't see any wearable loadad there)

![backpack-broken](https://user-images.githubusercontent.com/1031741/181349711-eaf42c7a-86a9-4008-b9cd-bc10e06dd67d.png)

## How to test the changes?
1. Go to: https://play.decentraland.zone/?renderer-branch=fix/unexpected-backpack-behaviours
2. Reproduce the steps described above.
3. Check the backpack works as expected and our COLLECTIONS are always being loaded correctly.
4. Check also the avatar preview is always being loaded with the correct equipped wearables.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
